src/arrays/referencing_array_and_its_elements/hard/q6/RightDownTheMiddle.java

```diff
@@ -0,0 +1,42 @@
+package arrays.referencing_array_and_its_elements.hard.q6;
+
+/*
+Write a program that takes an array of ten integers and prints a string containing the first half of the array elements.
```

Comment:
Change question text to: 
`Write a program that takes an array of ten integers and prints a string containing the first half of the array elements. To do this, create a method called splitHlaf that takes an integer array and returns the string representation of the first half of the array. For this program, assume the first half of the array is equal to array.length/2. Note that the returned string should have spaces between each number and a space at the end. Finally, print the returned string in the main method. Your method should be general enough to work with any size of an array and any size of an integer.`